frame/support/test/tests/issue2219.rs

```diff
@@ -184,7 +184,8 @@ frame_support::construct_runtime!(
 
 #[test]
 fn create_genesis_config() {
-	GenesisConfig {
+	let config = GenesisConfig {
 		module: module::GenesisConfig { request_life_time: 0, enable_storage_role: true },
 	};
+	drop(config);
```

Comment:
This test was weird, and became even more weird.

Reply:
let me switch the drop for a couple of asserts.